Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -146,6 +199,7 @@ public CsharpCompiler(ErrorReporter reporter)
 
     private void EmitInitNewArrays(BuiltIns builtIns, ConcreteSyntaxTree wr) {
       var dafnyNamespace = CreateModule("Dafny", false, false, null, wr);
+      AddMultiMatcher(dafnyNamespace);
```

Comment:
The name `EmitInitNewArrays` doesn't match this content. Consider moving the two statements

```csharp
var dafnyNamespace = CreateModule("Dafny", false, false, null, wr);
AddMultiMatcher(dafnyNamespace);
```

out of `EmitInitNewArrays` and passing `dafnyNamespace` to `EmitInitNewArrays`

Reply:
Done! I also moved `AddMultiMatcher`, now renamed `EmitMultiMatcher` to the `CsharpMockWriter` class.